keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -225,6 +225,51 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
+class NMSDecoder(tf.keras.layers.Layer):
```

Comment:
Should this be a public API?

Would it be possible to use the `non_max_supression` layer for this?

Reply:
Added a TODO. For now we don't have a TPU compatible layer, and the one we have is probably buggy given I cannot reproduce a good result